enrollment-server/src/test/java/com/wultra/app/enrollmentserver/impl/service/converter/MobileTokenConverterTest.java

```diff
@@ -592,6 +594,47 @@ void testConvertUiPreapprovalScanQrSuppressed() throws Exception {
         assertNull(ui.getPreApprovalScreen());
     }
 
+    @Test
+    void testCreateStringSubstitutorWithNullValue() throws MobileTokenConfigurationException {
+        final OperationDetailResponse operationDetail = createOperationDetailResponse();
+        final Map<String, String> params = new HashMap<>();
+        params.put("amount", "100.00");
+        params.put("currency", "EUR");
+        params.put("iban", null);
+        operationDetail.setParameters(params);
+
+        final OperationTemplateEntity operationTemplate = new OperationTemplateEntity();
+        operationTemplate.setAttributes("""
+                [
+                  {
+                    "id": "operation.amount",
+                    "type": "AMOUNT",
+                    "text": "Amount",
+                    "params": {
+                      "amount": "amount",
+                      "currency": "currency"
+                    }
+                  },
+                  {
+                    "id": "operation.account",
+                    "type": "KEY_VALUE",
+                    "text": "To Account",
+                    "params": {
+                      "value": "iban"
+                    }
+                  }
+                ]""");
+
+        final Operation operation = tested.convert(operationDetail, operationTemplate);
+        final List<Attribute> attributes = operation.getFormData().getAttributes();
+
+
+        assertNotNull(operation);
+        assertNotNull(attributes);
```

Comment:
To avoid NPE, it is needed to reorder.
```suggestion
        assertNotNull(operation);
        final List<Attribute> attributes = operation.getFormData().getAttributes();

        assertNotNull(attributes);
```